stylesheets/commons/Brackets.less

```diff
@@ -354,6 +354,10 @@
 	grid-column: span 5;
 	padding: 1px 5px;
 	text-align: center;
+
+	> .match-countdown-block {
+		font-size: 95%;
+	}
```

Comment:
Do we still need the 5% font size reduction? I think I rather have it removed unless it looks really strange without it

Reply:
Is it still needed? Is it used on the main page match ticker? If not, we can probably remove both?